tests/rptest/services/databricks_client.py

```diff
@@ -0,0 +1,452 @@
+# Copyright 2024 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import os
+import logging
+from typing import List, Optional, Any
+from databricks.sdk import WorkspaceClient
+from pyspark.sql import SparkSession
+
+
+class DatabricksClient:
```

Comment:
maybe it this should be moved to `tests/rptest/clients/...` instead of `.../services/`?